string drone_id
uint8 drone_state
uint8 docking_slot
float32 battery_percent
float32[4] gps_position
float32 yaw